example/src/main/java/org/geysermc/mcprotocollib/protocol/example/MinecraftProtocolTest.java

```diff
@@ -177,19 +176,21 @@ private static void status() {
     private static void login() {
         MinecraftProtocol protocol;
         if (VERIFY_USERS) {
+            StepFullJavaSession.FullJavaSession fullJavaSession;
             try {
-                AuthenticationService authService = new MojangAuthenticationService();
-                authService.setUsername(USERNAME);
-                authService.setPassword(PASSWORD);
-                authService.setProxy(AUTH_PROXY);
-                authService.login();
-
-                protocol = new MinecraftProtocol(authService.getSelectedProfile(), authService.getAccessToken());
-                log.info("Successfully authenticated user.");
-            } catch (RequestException e) {
-                log.error("Failed to authenticate user.", e);
-                return;
+                fullJavaSession = MinecraftAuth.JAVA_CREDENTIALS_LOGIN.getFromInput(
+                        MinecraftAuth.createHttpClient(),
+                        new StepCredentialsMsaCode.MsaCredentials(USERNAME, PASSWORD));
+            } catch (Exception e) {
+                throw new RuntimeException(e);
             }
+
+            var mcProfile = fullJavaSession.getMcProfile();
+            var mcToken = mcProfile.getMcToken();
```

Comment:
We do not use var anywhere else in the code base.

Reply:
Ah yeah I see, took this from my SoulFire codebase. I exclusively use var for a few months and i copy-pasted that.